### ISSUE TITLE
Challenge Importing `MCLP` 

### ISSUE THREAD

**Issue by tmickleydoyle** [pysal/spopt#195]
I am seeing an error when trying to import `MCLP` from the spopt package for Python 3.8. Thank you 🙇 

```bash
(.venv) 2021-09-15 10:59:46 Desktop: pip install spopt
Collecting spopt
  Using cached spopt-0.1.2-py3-none-any.whl (64 kB)
Requirement already satisfied: pandas>=1 in ./.venv/lib/python3.8/site-packages (from spopt) (1.2.5)
Requirement already satisfied: scikit-learn>=0.22 in ./.venv/lib/python3.8/site-packages (from spopt) (0.24.2)
Requirement already satisfied: geopandas>=0.7 in ./.venv/lib/python3.8/site-packages (from spopt) (0.9.0)
Requirement already satisfied: networkx in ./.venv/lib/python3.8/site-packages (from spopt) (2.6.3)
Requirement already satisfied: pulp in ./.venv/lib/python3.8/site-packages (from spopt) (2.5.0)
Requirement already satisfied: scipy>=0.11 in ./.venv/lib/python3.8/site-packages (from spopt) (1.7.0)
Requirement already satisfied: numpy>=1.3 in ./.venv/lib/python3.8/site-packages (from spopt) (1.21.0)
Requirement already satisfied: libpysal in ./.venv/lib/python3.8/site-packages (from spopt) (4.5.1)
Requirement already satisfied: fiona>=1.8 in ./.venv/lib/python3.8/site-packages (from geopandas>=0.7->spopt) (1.8.20)
Requirement already satisfied: shapely>=1.6 in ./.venv/lib/python3.8/site-packages (from geopandas>=0.7->spopt) (1.7.1)
Requirement already satisfied: pyproj>=2.2.0 in ./.venv/lib/python3.8/site-packages (from geopandas>=0.7->spopt) (3.2.0)
Requirement already satisfied: attrs>=17 in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (21.2.0)
Requirement already satisfied: setuptools in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (41.2.0)
Requirement already satisfied: six>=1.7 in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (1.16.0)
Requirement already satisfied: cligj>=0.5 in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (0.7.2)
Requirement already satisfied: certifi in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (2021.5.30)
Requirement already satisfied: munch in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (2.5.0)
Requirement already satisfied: click-plugins>=1.0 in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (1.1.1)
Requirement already satisfied: click>=4.0 in ./.venv/lib/python3.8/site-packages (from fiona>=1.8->geopandas>=0.7->spopt) (8.0.1)
Requirement already satisfied: pytz>=2017.3 in ./.venv/lib/python3.8/site-packages (from pandas>=1->spopt) (2021.1)
Requirement already satisfied: python-dateutil>=2.7.3 in ./.venv/lib/python3.8/site-packages (from pandas>=1->spopt) (2.8.1)
Requirement already satisfied: threadpoolctl>=2.0.0 in ./.venv/lib/python3.8/site-packages (from scikit-learn>=0.22->spopt) (2.2.0)
Requirement already satisfied: joblib>=0.11 in ./.venv/lib/python3.8/site-packages (from scikit-learn>=0.22->spopt) (1.0.1)
Requirement already satisfied: requests in ./.venv/lib/python3.8/site-packages (from libpysal->spopt) (2.26.0)
Requirement already satisfied: beautifulsoup4 in ./.venv/lib/python3.8/site-packages (from libpysal->spopt) (4.10.0)
Requirement already satisfied: jinja2 in ./.venv/lib/python3.8/site-packages (from libpysal->spopt) (3.0.1)
Requirement already satisfied: soupsieve>1.2 in ./.venv/lib/python3.8/site-packages (from beautifulsoup4->libpysal->spopt) (2.2.1)
Requirement already satisfied: MarkupSafe>=2.0 in ./.venv/lib/python3.8/site-packages (from jinja2->libpysal->spopt) (2.0.1)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./.venv/lib/python3.8/site-packages (from requests->libpysal->spopt) (1.26.6)
Requirement already satisfied: idna<4,>=2.5 in ./.venv/lib/python3.8/site-packages (from requests->libpysal->spopt) (3.2)
Requirement already satisfied: charset-normalizer~=2.0.0 in ./.venv/lib/python3.8/site-packages (from requests->libpysal->spopt) (2.0.5)
Installing collected packages: spopt
Successfully installed spopt-0.1.2
(.venv) 2021-09-15 10:59:55 Desktop: python --version
Python 3.8.0
(.venv) 2021-09-15 11:00:03 Desktop: python
Python 3.8.0 (default, May  7 2021, 13:40:59) 
[Clang 11.0.0 (clang-1100.0.33.12)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from spopt.locate.coverage import MCLP

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'spopt.locate'
>>> 
```

Reference: https://github.com/openjournals/joss-reviews/issues/3330

**Comment by jGaboardi** [comment]
The `locate` module has been merged into the code base, but it is not yet part of a release. However, It will be in the next release.

**Comment by tmickleydoyle** [comment]
Great! I think I used a code sample from the README file, but I will check again. Thank you for the prompt response.

**Comment by jGaboardi** [comment]
@tmickleydoyle This is solved with in [`v0.2.1`](https://github.com/pysal/spopt/releases/tag/v0.2.1).
